concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -441,18 +447,24 @@ jobs:
       - get: pipeline-trigger
         passed: [ 'deploy']
         trigger: true
+      - get: cf-release
+        params:
+          submodules:
+          - src/smoke-tests
       - get: paas-cf
         passed: ['deploy']
       - get: cf-manifest
         passed: ['deploy']
-        trigger: true
       - get: bosh-secrets
```

Comment:
I don't think this uses `bosh-secrets` any more (neither do the acceptance tests).